### ISSUE TITLE
WebUI Websocket always disconnected

### ISSUE THREAD

**Issue by ReinforceZwei** [airsonic-advanced/airsonic-advanced#959]
## Problem description

WebUI always shows disconnected
![image](https://user-images.githubusercontent.com/39955851/200268915-cad74a6f-f42c-495b-b59e-3f5ca0c9c74a.png)

### Steps to reproduce

_I'm not sure how to reproduce it as it suddenly stop working when I use it today._

_Already using latest version and the problem exists_
1. Start airsonic with `java -Dserver.port=8800 -Dserver.servlet.context-path=/music -jar airsonic.war`
2. Login to WebUI
3. It won't connect
4. Browser Dev Tool shows errors
![image](https://user-images.githubusercontent.com/39955851/200270046-26531e13-f490-4e62-944d-e6ae958b9ff6.png)
5. Airsonic output error log (see below)

## System information

 * **Airsonic version**: Was using [this version](https://github.com/airsonic-advanced/airsonic-advanced/releases/tag/11.0.0-SNAPSHOT.20220320083637) and had above issue, updated to [latest](https://github.com/airsonic-advanced/airsonic-advanced/releases/tag/11.0.0-SNAPSHOT.20220625052932) still 
 * **Operating system**: Ubuntu 22, `Linux RF-NUC 5.15.0-48-generic #54-Ubuntu SMP Fri Aug 26 13:26:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`
 * **Java version**: 
 ```
openjdk version "11.0.16" 2022-07-19
OpenJDK Runtime Environment (build 11.0.16+8-post-Ubuntu-0ubuntu122.04)
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Ubuntu-0ubuntu122.04, mixed mode, sharing)
```
 * **Proxy server**: None
 * **Client**: Chrome and Firefox
 * **Language**: English

## Additional notes

Error logs from Airsonic: https://pastebin.mozilla.org/3SnqZLa9 (expire in 3 weeks)
https://reinforce.moe/privatebin/?f9f8e497f7c29970#V8tZ8qevwMrNvAddbSCjCiekqrY1eZfVRXt2gZAT51M (backup, never expire)

**Comment by verybadsoldier** [comment]
I have the same issue. Websocket connection not working after a fresh install of latest WAR file. Running on Win10 without proxy. Looks pretty much the same as described in this ticket so I won't attach additional logs (if there is no special need for it).
Any ideas yet regarding this?

**Comment by ReinforceZwei** [comment]
> I have the same issue. Websocket connection not working after a fresh install of latest WAR file. Running on Win10 without proxy. Looks pretty much the same as described in this ticket so I won't attach additional logs (if there is no special need for it). Any ideas yet regarding this?

I switched to docker installation and it works fine.

**Comment by verybadsoldier** [comment]
Yes I am coming from docker and was looking to run it natively for performance reasons.

I just tested the latest version from this fork and replaced my WAR and it instantly worked:
https://github.com/kagemomiji/airsonic-advanced
